book/chapters/chapter2/data_and_basic_modeling.qmd

```diff
@@ -413,14 +413,14 @@ In our decision tree example, we can infer from the `ParamSet` output that:
 * `keep_model` must be a "logical" (`ParamLgl`) taking values `TRUE` or `FALSE` with default `FALSE`
 * `xval` must be an "integer" (`ParamInt`) taking values between `0` and `Inf` with a default of `10` and has a set value of `0`.
 
-In rare cases (we try to minimize it as much as possible), we alter hyperparameter values when constructing a learner.
-When we do this, the reason will always be given in the learner help page.
-In the case of `lrn("regr.rpart")`, we changed the `xval` default to `0` because `xval` controls internal cross-validations and if a user accidentally leaves this at the default `10`, model training can take a long time.
+In rare cases (`mlr3` tries to minimize it as much as possible), hyperparameters are initialized to values which deviate from the default in the underlying package.
```

Comment:
I think an active voice makes more sense. It isn't `mlr3` that decides on the changes it is us making an active decision. Currently it sounds like software makes this decision